gatsby/content/projects/bridges/gupshup-matrix.mdx

```diff
@@ -0,0 +1,31 @@
+---
+layout: project
+title: gupshup-matrix
+categories:
+ - bridge
+description: A WhatsApp Matrix-Gupshup bridge
+author: Alejandro Herrera
+maturity: Beta
+language: Python
+license: MIT
+repo: https://github.com/bramenn/gupshup
+room: "#gupshup-bridge:matrix.org"
+featured: true
```

Comment:
```suggestion
featured: false
```

We usually add bridges to the featured list once they are more mature :)

Reply:
Understood